src/Resources/contao/widgets/ListWizard.php

```diff
@@ -115,117 +118,21 @@ public function generate()
 	 * @param DataContainer $dc
 	 *
 	 * @return string
+	 *
+	 * @throws \Exception
+	 * @throws \Contao\CoreBundle\Exception\RedirectResponseException
+	 *
+	 * @deprecated Deprecated since Contao 4.3 to be removed in 5.0. Use the BackendCsvImportController instead.
```

Comment:
Changed in https://github.com/contao/core-bundle/pull/444/commits/7ba5e323248780ad1d729674c7de0cce3d1677af